docs/general-usage/decorators.rst

```diff
@@ -174,6 +174,10 @@ You can now query your adverts with the following query:
         }
     }
 
+The default ``per_page`` value is 10 and can be changed with the ``GRAPPLE_PAGE_SIZE`` setting.
```

Comment:
suggestion: looks like it may be time to move to a single settings approach. Django Rest Framework and others do it. Wagtail itself will eventually move to that as well.

With these we now have (from the example settings):

```python
GRAPPLE_APPS = {"images": "", "home": "", "documents": ""}
GRAPPLE_ADD_SEARCH_HIT = True
GRAPPLE_PAGE_SIZE = 10
GRAPPLE_PAGE_SIZE_MAX = 50
```

Would look so much better as:
```python
GRAPPLE = {
    "apps": {"images": "", "home": "", "documents": ""},
    "add_search_hit": True,
    "page_size": 10,
    "page_size_max": 50,   # `max_page_size` ?
}
```

would be amazing if we could support a list for Grapple apps (i.e. `["images", "home", "documents"]`) as well -- the current dict with empty values is a long bugbear of mine -- but not imperative on this PR

Reply:
Makes sense, I'll update the code accordingly.

Reply:
🌟

Reply:
@zerolab updated to use single object settings approach!

Reply:
I just added a `settings` module to easily manage all Grapple settings. It's inspired by Django Rest Framework's settings module. 👀